cli/Cargo.toml

```diff
@@ -60,12 +60,14 @@ experimental = [
     # The following features are experimental:
     "command",
     "playlist",
-    "workload"
+    "workload",
+    "workload-smallbank"
 ]
 
 command = ["cylinder", "hex", "protobuf", "reqwest", "serde", "serde_json"]
 playlist = ["cylinder", "serde_json"]
-workload = ["ctrlc", "cylinder", "rand", "transact/family-command-workload", "transact/family-smallbank-workload"]
+workload = ["ctrlc", "cylinder", "rand", "transact/family-command-workload"]
```

Comment:
Should there also be a "workload-command" feature? Otherwise, getting smallbank also gets you command family

Reply:
Based on discussion - Leaving as is with the intention of removing the "workload-smallbank" feature after it's stabilized